packages/core/src/generator/index.ts

```diff
@@ -98,6 +98,10 @@ export class UnoGenerator<Theme extends object = object> {
     raw: string,
     alias?: string,
   ): Promise<StringifiedUtil<Theme>[] | undefined | null> {
+    // eslint-disable-next-line regexp/no-unused-capturing-group
+    if (/(^\(|\)$)/.test(raw))
+      raw = raw.replace(/(^\(|\)$)/g, '')
```

Comment:
This is too specific for core. Core shouldn't aware of this and shouldn't do modifications to `raw`